packages/wdio-sync/tests/executeHooksWithArgs.test.js

```diff
@@ -0,0 +1,38 @@
+import executeHooksWithArgs from '../src/executeHooksWithArgs'
+
+describe('executeHooksWithArgs', () => {
+    it('multiple hooks, multiple args', async () => {
+        const hookHoge = () => {return 'hoge'}
+        const hookFuga = () => {return 'fuga'}
+        const argHoge = { hoge: 'hoge' }
+        const argFuga = { fuga: 'fuga' }
+        const res = await executeHooksWithArgs([hookHoge, hookFuga], [argHoge, argFuga])
+        expect(res).toHaveLength(2)
+        expect(res).toContain('hoge')
```

Comment:
Running contain here 2 times doesn't make much sense. Can you please change contain to equal to compare res with result array